pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1225,7 +1225,7 @@ func (c *Cloud) getMountDevice(
 	if deviceAllocator == nil {
 		// we want device names with two significant characters, starting with
 		// /dev/xvdba (leaving xvda - xvdz and xvdaa-xvdaz to the system)
```

Comment:
Would be good to update this comment per https://github.com/kubernetes/kubernetes/pull/41455.  xvdaa - xvdaz are not allowed